backend/apps/issue_categorization/services/signals.py

```
@@ -0,0 +1,62 @@
+        return
+    
+    try:
+        engine = CategorizationEngine()
+        suggestion = engine.auto_categorize_issue(
+            instance,
+            instance.title,
+            instance.description,
+            getattr(instance, 'body', '')
+        )
+        logger.info(f"Auto-categorized issue {instance.id}: {suggestion.id}")
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)